SPECS/ksh/ksh.spec

```diff
@@ -37,6 +30,9 @@ Requires(post): grep
 Requires(post): systemd
 
 Requires(postun): sed
+Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1
+Requires(preun): /bin/sh
+Requires(verify): /bin/sh
```

Comment:
I do not think we need to specify these Requires.

Reply:
Working on the tests in this package, will put this PR into draft